### ISSUE TITLE
Adding changes back to Emacs tree

### ISSUE THREAD

**Issue by DamienCassou** [DamienCassou/auth-password-store#60]
I would like to backport the changes you guys did to the Emacs tree as auth-password-store/auth-source-pass is there too. Did you sign the GNU agreement? If not, are you willing to?

- [x] @jedrz 
- [x] @dieggsy 
- [x] @jabranham 
- [x] @jellelicht

**Comment by jabranham** [comment]
I've signed the papers already

**Comment by jellelicht** [comment]
I did not, but am willing to. Is there a link somewhere which describes the general procedure?

**Comment by DamienCassou** [comment]
> I did not, but am willing to. Is there a link somewhere which describes the general procedure?

@jwiegley wrote [something like that previously](https://github.com/jwiegley/use-package/issues/282):
> Just send an e-mail to assign@gnu.org. Tell them either that you'd like to sign a copyright assignment for contributions to [auth-source-pass] to be included in future versions of Emacs; or say that you'd like to sign a general assignment form for any future contributions to Emacs. They will send you the PDF that you need to sign and return to them.

You might also need to tell your country of residence.

**Comment by jedrz** [comment]
I did not though I am going to contact the FSF. I will inform you about any progress.

**Comment by DamienCassou** [comment]
Any news @jedrz, @dieggsy and @jellelicht?

**Comment by jedrz** [comment]
I sent the agreement signed by me and now I am waiting for receiving the copy signed by FSF back.

**Comment by jellelicht** [comment]
Likewise for me.

**Comment by dieggsy** [comment]
@DamienCassou ah, sorry about that, I'll send out the email today

**Comment by jellelicht** [comment]
@DamienCassou  I'm just waiting on getting the signature back from the FSF, so afaik everything is okay from my end. Sorry for the delay!

**Comment by jellelicht** [comment]
Everything is taken care of, thanks for being patient.

**Comment by DamienCassou** [comment]
@jellelicht thank you for following the process and telling me about it.

**Comment by DamienCassou** [comment]
@jedrz @dieggsy what are the news?

**Comment by jedrz** [comment]
Yes, my assignment should be ready within couple of days.

**Comment by dieggsy** [comment]
@DamienCassou fell behind on this, sorry. I'm in contact with the FSF and will fill out my forms within the next couple days, depending on when they send them over to me

**Comment by jedrz** [comment]
@DamienCassou my assignment is completed.

**Comment by dieggsy** [comment]
@DamienCassou I've sent in my papers, but I didn't receive a reply from the FSF yet

**Comment by dieggsy** [comment]
@DamienCassou Just received word that my copyright assignment is now complete

**Comment by DamienCassou** [comment]
Thank you all. I will soon proceed to send all our changes to the Emacs tree.

**Comment by DamienCassou** [comment]
I sent a series of patch with your names to the Emacs developers.

**Comment by DamienCassou** [comment]
The patches have been merged into `master`.
